### DISCUSSION TITLE
better preprocessor modells for normal and depth maps

### DISCUSSION THREAD

**Discussion by gsgoldma** [Mikubill/sd-webui-controlnet#504] [General]
https://github.com/hack-mans/clipdrop-api

is there any way for this to be integrated or is it not compatible?

**Comment by ljleb** [comment]
It should work, but maybe better to put this in its own extension? As it is a paid service... not sure.

**Reply by ljleb** [reply]
There is a maintained api to communicate with controlnet now, you could use it to send detectmaps to specific control units.

**Reply by ljleb** [reply]
maybe this would have its place in the depth library extension? https://github.com/jexom/sd-webui-depth-lib
